### PR TITLE
Add TF check for Task Def having different Execution and Task roles

### PR DESCRIPTION
**Pull request by MooreDerek**
New check as discussed in #2602

### CODE REVIEW

File: checkov/terraform/checks/resource/aws/ECSTaskDefinitionRoleCheck.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ECSTaskDefinitionRoleCheck(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that the Execution Role ARN and the Task Role ARN are different in ECS Task definitions"
+        id = "CKV2_AWS_37"
```
Comment (gruebel) [inline checkov/terraform/checks/resource/aws/ECSTaskDefinitionRoleCheck.py:8]:
```suggestion
        id = "CKV_AWS_245"
```
we only use `CKV2_` for YAML checks

**Review by gruebel — COMMENTED** [review]
please just adjust the ID, the failing int tests has nothing to do with your changes, so no worries 😄

**Review by gruebel — APPROVED** [review]
thanks 🥇

**Comment by schosterbarak** [comment]
Thanks @MooreDerek ! can you please apply for swag at: https://docs.google.com/forms/d/e/1FAIpQLSe6ZzxKMY8PPcPjwUR57c0VmaI1Y2Ib9GtJwVxQ40h9jiTu9w/viewform ? 
thanks :)

**Comment by MooreDerek** [comment]
@schosterbarak  Permission denied :(

**Comment by schosterbarak** [comment]
Sorry @MooreDerek can you send me a T-Shirt size and address to [bschostergoi@paloaltonetworks.com](mailto:bschostergoi@paloaltonetworks.com)

**Pull request merged**